packages/adaptive_scaffold/lib/src/slot_layout.dart

```diff
@@ -0,0 +1,80 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+import 'breakpoints.dart';
+import 'slot_layout_config.dart';
+
+/// A Widget that takes a mapping of [SlotLayoutConfig]s to breakpoints and
```

Comment:
Same as `SlotLayoutConfig`: maybe this shouldn't be a widget to avoid misuse. Is there any occasion where this class will be displayed outside of `AdaptiveLayout`?

Reply:
SlotLayout can be used outside AdaptiveLayout without losing any functionality. I assume if someone wanted to create a custom implementation of AdaptiveLayout they would be able to use SlotLayout for their slots.